app/views/content/is-teaching-right-for-me/get-school-experience.md

```diff
@@ -27,6 +27,10 @@ quote:
     text: "I volunteered at a local primary school for 2 days a week and from the moment I walked into my first class, it felt very natural to be interacting with children."
     name: "Zainab, career changer"
     classes: quote--indent
+expander:
+  non-uk-get-school-experience:
+    title: get school experience
+    text: To get school experience in England, you'll need the right to work in the UK. This may be difficult if you have not started your training yet. However, you do not need classroom experience to train to teach in England.  
```

Comment:
```suggestion
    text: To request a placement in England through our get school experience service, you'll need the right to work in the UK. This may be difficult if you have not started your training yet. However, you do not need classroom experience to train to teach in England.  
```

does the right to work in England criteria just apply to our GSE service (I think probably)?

In which case, I would suggest rewording this (please reword as needed!)

Reply:
right to work also applied to internships as well